packages/core/addon/components/field-renderer.js

```diff
@@ -3,16 +3,32 @@ import { dasherize } from '@ember/string';
 import { action } from '@ember/object';
 import { tracked } from '@glimmer/tracking';
 
-// TODO This will be part of the official API. Move this into core as it solidifies
+const defaultSchemaAttrs = [
+  'title',
+  'type',
+  'is-meta',
+  'name',
+  'embedded'
+];
+
+let renderNonce = 0;
 export default class FieldRenderer extends Component {
   @tracked newFieldName;
+  @tracked newFieldLabel;
   @tracked currentNonce;
+  @tracked renderNonce;
 
   constructor(...args) {
     super(...args);
 
     this.newFieldName = this.args.field.name;
+    this.newFieldLabel = this.args.field.label;
```

Comment:
@robustdj here we are initializing the property that will hold the local label's input field state. we bind the input field to this property instead of directly to the `Field.label` because changing `Field.label` requires actually calling `Field.setLabel()`.